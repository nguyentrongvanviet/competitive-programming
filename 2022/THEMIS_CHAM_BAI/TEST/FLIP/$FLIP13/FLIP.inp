136 644
152315 318627
679229 286277
197456 36634
709237 300837
793287 969587
516558 417271
744674 479848
540700 93594
802465 998932
584788 238661
476025 842621
472086 238542
970269 122347
565095 733924
709065 679808
848537 563625
651284 351320
749507 106365
963265 703817
732199 217612
519007 864871
290694 599472
622948 14361
460479 669374
300729 842579
908852 57780
282657 549489
203266 452796
206134 609665
970137 133476
757414 525291
259802 637330
664684 79725
231329 39649
671742 157742
59446 91529
261366 617570
131389 556965
588782 202135
649 171088
563459 145685
672376 233858
343314 401887
799811 294037
549059 695469
799156 591910
474365 317505
103752 758710
606938 570083
916702 586980
394413 245528
893464 156656
377535 633740
324723 320448
422039 247115
546222 407699
637704 319814
242001 863037
957708 425073
77483 67834
65235 471578
35474 744464
305192 708752
744097 176440
777001 370610
196959 573662
974852 179783
134875 741406
721493 375440
165574 668032
510248 994995
539912 330492
772606 298238
516243 875488
791356 956355
721267 964465
12523 333186
532743 307484
376986 217899
484471 673517
489234 11112
47004 826278
6792 779032
733883 437264
688325 511190
387855 830642
31696 408289
450476 963561
203853 325653
201674 70550
15975 383541
927931 153526
904869 44584
974078 998272
342492 595175
315949 949376
271644 418747
775337 187172
481990 19578
376374 742756
307089 760143
583455 414789
696890 952708
247633 37405
855377 850050
853017 533063
667873 571258
303971 932638
804070 566713
63547 450602
457471 540698
553823 167659
111659 194244
108759 167352
850995 576628
15402 96623
233976 586892
357770 319276
734615 941939
747299 251920
784606 890023
831232 848488
217084 189012
868811 789262
287877 65688
508824 172530
303634 649767
934436 730781
544248 118090
431891 994424
776874 335645
289851 617796
807428 702787
756905 96882
45586 84173
85253 860794
979506
652682
322309
21280
267714
473904
335826
771747
87321
298306
420382
433602
909911
651576
964335
568090
469758
600409
2856
790600
19178
319336
555816
601273
776788
274099
897309
671948
508432
806057
547275
462293
709440
730810
956747
187412
161969
646111
448292
914894
77701
41191
31851
150320
866538
666537
537166
225118
795228
752292
227903
95923
238207
997031
609110
676389
476823
620085
904734
894702
542767
431513
581909
655558
227061
567786
536892
964270
361259
821811
35631
799465
504042
483837
831991
311130
353976
193706
903131
199724
349193
386496
593415
614214
339242
816285
213833
437809
177381
373155
21652
1801
255003
503017
397793
937075
396318
613710
228889
943723
185660
488293
994984
175925
217242
253303
416879
903466
927439
788087
468157
824038
59820
458917
661340
586479
166084
585436
659437
871544
405426
338706
682503
990865
917062
365229
953174
745699
17836
810553
875464
998550
552341
974692
29259
276479
218927
304908
773157
597778
390416
701302
614637
129642
785395
962402
719347
101291
504759
98982
990623
889234
195750
114112
85417
322845
594957
653275
268655
289354
576163
549118
956013
269268
284099
437933
555546
508933
996875
21793
943260
218152
197796
693227
492199
618549
509662
856938
64779
666155
366605
483359
821728
990861
66041
126631
514484
361699
988326
188070
227725
210024
552101
103383
79095
800482
395348
666707
737034
437641
994060
756293
735066
579667
481498
486195
217505
887086
956669
461466
482727
512817
916871
677135
281853
885540
511810
121991
207347
202982
490013
743007
630585
441252
903598
946740
260970
996176
459512
721164
447497
900891
642219
334622
89784
583298
196574
265572
6139
442755
88615
616495
261309
349881
517822
269864
828608
453115
259470
901320
832799
85069
583550
62425
676443
977730
552639
560490
527516
587779
21673
497896
63196
895544
846999
549166
120885
247285
218408
654048
970727
984850
616074
888164
230936
313954
758254
49287
58674
211542
818521
469302
752874
972473
958044
65273
998631
571356
520709
818946
157037
126848
395352
463099
549816
239195
684798
630700
100682
201661
511584
477507
57633
226132
81785
300932
890157
979754
90435
57220
789677
172161
981099
380861
323733
702035
359186
197754
96009
530960
510923
205196
312050
738443
458300
485701
822420
726515
753014
161608
985949
371580
73655
538299
593641
260522
526781
833075
988395
215768
875083
311693
493201
114751
570664
247577
895709
172579
6823
217988
723460
381134
600055
349859
755151
156382
729247
584671
78527
561041
419119
239053
997379
818546
989144
18874
468635
387363
243123
766288
91316
104085
915251
597107
921848
280179
806722
186209
143875
599200
105154
975358
374923
85997
636900
470332
947738
450579
519718
192769
817156
91945
695754
965371
79065
526979
236127
781430
792082
443201
514128
84854
857566
298448
541735
33949
414264
58494
244374
57494
847533
126760
457451
891783
914981
691853
266822
419846
800259
209525
341245
739189
386174
48182
979502
290293
991020
46213
169399
383213
282654
996058
952231
675529
575444
712308
446494
348096
296
151132
457506
283506
170412
321517
431951
906258
970576
516522
292985
174969
148208
21644
32919
299580
389108
934326
731547
67368
784151
91487
62754
184439
805816
443103
721641
882908
939269
976212
267772
101857
34298
522285
740094
572620
447128
654732
891908
684400
400726
884752
518905
23827
836472
439464
177811
300503
436934
757968
150122
661452
597111
153243
441307
638327
173131
239009
574720
843393
382336
848775
409170
872241
590422
695559
247151
860325
877386
772612
358363
975838
693580
752578
288269
258559
110859
676289
744514
274289
83148
491238
916996
968549
981165
725239
451521
728448
213691
569028
605017
6881
905887
93945
961241
908673
264727
331909
226211
323111
493226
47072
189348
952735
265044
2658
189672
820110
811689
735834
548514
309651
150056
45412
484157
591657
408911
923087
181924
419037
705228
491367
609489
749120
567821
575969
228040
653646
963400
345266
475371
111179
295647
463818
289989
17641
645466
869912
132914
358356
257426
345786
738329
891736
348854
215242
567738
715618
437452
712235
51696
339888
75910
570571
800810
117695
542299
948902
76213
695450
867904
815239
172627
180692
969660
583461
190535
448368
988360
379617
445619
715997
585506
482922
53491
751646
931509
803358
34442
866571
204051
601562
386492
85777
725721
745458
24139
769145
226974
129880
349727
560411
464986
887375
298852
136386
205300
374681
150602
859231
394712
716394
685877
627011
492678
924678
