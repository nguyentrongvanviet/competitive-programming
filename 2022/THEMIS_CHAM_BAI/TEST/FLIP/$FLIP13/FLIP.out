318627 286277 36634 300837 969587 417271 479848 93594 998932 238661 476025 238542 970269 565095 679808 563625 351320 106365 963265 217612 519007 290694 14361 460479 300729 57780 282657 203266 206134 970137 525291 259802 79725 39649 157742 59446 261366 131389 202135 649 145685 233858 343314 294037 549059 591910 317505 103752 570083 586980 245528 156656 377535 320448 247115 407699 319814 242001 957708 77483 65235 35474 305192 176440 370610 196959 974852 134875 375440 165574 994995 330492 298238 516243 956355 964465 12523 307484 217899 484471 11112 47004 6792 437264 511190 387855 31696 963561 325653 70550 15975 927931 44584 998272 342492 949376 271644 187172 19578 376374 307089 414789 952708 37405 855377 533063 571258 932638 566713 63547 457471 167659 111659 108759 576628 15402 233976 319276 941939 251920 784606 831232 217084 789262 65688 172530 303634 934436 118090 994424 335645 289851 702787 96882 45586 85253 